pt/endpoints/bid/resource.py

```diff
@@ -185,7 +185,7 @@ def get(self):
                     tender.uuid
                     for tender in Tenders.query.filter(
                         Tenders.user_id == g.uuid,
-                        Tenders.start_time >= datetime.utcnow(),
+                        Tenders.start_time >= datetime.today(),
```

Comment:
Maybe we need to add a comment to explain why we use .today here

Reply:
Add description to README in d1e8a71.
This is my negligence.